components/playground/playground.go

```diff
@@ -395,7 +395,9 @@ func (p *Playground) bootCluster(options *bootOptions) error {
 	}{}
 
 	var monitorCmd *exec.Cmd
+	var grafana *grafana
 	if options.monitor {
```

Comment:
It's better to run Prometheus and Grafana by default in our new playground.

Reply:
currently, --monitor control both Prometheus and Grafana, if we want to run it separately we need to add more flag options.
I think we can change --monitor to default true?
I will do this and change the subcommand scale-in/scale-out not hidden when all these things are done if it's ok to make it as true default.